gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
Shouldn't this be a patch version?

Reply:
> Shouldn't this be a patch version?

Yeah it should: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/aac0a6d958e152b02a2c6080a060e75e9cb52bd2

Reply:
> Yeah it should: aac0a6d

Not .1?

Reply:
> > Yeah it should: [aac0a6d](https://github.com/sonatype/sonatype-react-shared-components/commit/aac0a6d958e152b02a2c6080a060e75e9cb52bd2)
> 
> Not .1?

You suggested patch instead of minor version so that means I back up from 2.6.0 to 2.5.x, then increment up 0.0.1 from there

Reply:
> You suggested patch instead of minor version so that means I back up from 2.6.0 to 2.5.x, then increment up 0.0.1 from there

But you went up by 0.0.2:
![image](https://user-images.githubusercontent.com/3630091/97352812-7fd17c00-1869-11eb-89a7-bed8f7536a2c.png)

Reply:
> > You suggested patch instead of minor version so that means I back up from 2.6.0 to 2.5.x, then increment up 0.0.1 from there
> 
> But you went up by 0.0.2:

Hmmm, odd, merged from Master and bumped again: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/3e9ac67844b18cbbbb5b4912d04cec8e5220863f